modules/integrate-circle-ci.adoc

```diff
@@ -35,19 +35,25 @@ jobs:
           password: $DOCKERHUB_PASSWORD
     steps:
       - checkout
+      - run:
+          name: Create output directory for checks results
+          command: |
+              mkdir junit
       - run:
           name: Install roxctl
           command: |
               curl -H "Authorization: Bearer $ROX_API_TOKEN" https://$STACKROX_CENTRAL_HOST:443/api/cli/download/roxctl-linux -o roxctl && chmod +x ./roxctl
       - run:
           name: Scan images for policy deviations and vulnerabilities
           command: |
-              ./roxctl image check --endpoint "$STACKROX_CENTRAL_HOST:443" --image "<your_registry/repo/image_name>" <1>
+              ./roxctl image check --endpoint "$STACKROX_CENTRAL_HOST:443" --image "<your_registry/repo/image_name>" -o junit > junit/image.xml <1>
       - run:
           name: Scan deployment files for policy deviations
           command: |
-              ./roxctl image check --endpoint "$STACKROX_CENTRAL_HOST:443" --image "<your_deployment_file>" <2>
+              ./roxctl deployment check --endpoint "$STACKROX_CENTRAL_HOST:443" --file "<your_deployment_file>" -o junit > junit/deployment.xml <2>
```

Comment:
Does the callout need to be changed? Seems like there's new stuff going on here.